.github/workflows/backstage-release.yaml

```diff
@@ -127,19 +127,17 @@ jobs:
 
       - name: Post status to Slack
         # Note: using env.SLACK_WEBHOOK here because secrets are not allowed in the if block.
-        if: env.SLACK_WEBHOOK != ''
-        uses: rtCamp/action-slack-notify@v2.3.0
+        if: ${{ env.SLACK_WEBHOOK != '' && failure()}}
+        uses: rtCamp/action-slack-notify@v2.1.0
```

Comment:
why are we downgrading the action?

Reply:
I wanted to use the same version with a working setup.